SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/RPCPayload.h"
+#include "Schema/UnrealObjectRef.h"
+#include "SpatialConstants.h"
+
+#include "UObject/WeakObjectPtr.h"
+#include "CoreMinimal.h"
+
+struct FUnrealObjectRef;
```

Comment:
I don't think we can forward declare this and it looks like it's being included anyway.

Reply:
Removed.